package.json

```diff
@@ -54,7 +54,9 @@
     "relay-runtime": "https://github.com/alloy/relay/releases/download/v1.5.0-artsy.5/relay-runtime-1.5.0-artsy.5.tgz",
     "react": "16.8.2",
     "react-relay": "https://github.com/alloy/relay/releases/download/v1.5.0-artsy.5/react-relay-1.5.0-artsy.5.tgz",
-    "react-test-renderer": "16.8.2"
+    "react-test-renderer": "16.8.2",
+    "prop-types": "15.7.2",
+    "@babel/runtime": "7.1.5"
```

Comment:
@zephraph - would you mind documenting your approach in the PR description as you go along? I.e., 
> To remove bundle duplicates, I first noticed x, then added y to the resolutions field. This then forces a particular dep to resolve to a specific version in the parent app (and so on)

Then we'll be able to come up with better instructions for devs not familiar with the lower level details.